snooty/rstparser.py

```diff
@@ -1072,29 +1072,23 @@ def register_spec_with_docutils(
         )
         builder.add_directive(name, DocutilsDirective)
 
+    # copy tabs directive declaration
```

Comment:
This is referencing it, not copying it. Comments should also explain why you're doing this, if possible